src/theme.css

```diff
@@ -0,0 +1,28 @@
+:root {
+  /* Colors */
+  --color-neutral-1: #ffffff;
+  --color-neutral-2: #f4f4f4;
+  --color-neutral-3: #ebebeb;
+  --color-neutral-4: #d6d6d6;
+  --color-neutral-5: #adadad;
+  --color-neutral-6: #7a7a7a;
+  --color-neutral-7: #4d4d4d;
+  --color-neutral-8: #2e2e2e;
+  --color-neutral-9: #000000;
+
+  --color-primary-1: #f4f4f4;
```

Comment:
I understand numbers in our own color variables. But variables exposed to theming shouldn't have numbers so they are easier to understand